NearbyMessagesDevices/build.gradle

```diff
@@ -3,6 +3,10 @@
 buildscript {
     repositories {
         jcenter()
+        maven {
```

Comment:
Why not just `google()` ?

Reply:
+1, I know this was a problem with older versions of Studio, but you should be set to use google() now. :)